src/backend/distributed/commands/sequence.c

```diff
@@ -178,6 +179,12 @@ ExtractColumnsOwningSequences(Oid relationId, List **columnNameList,
 		}
 
 		char *columnName = NameStr(attributeForm->attname);
+		if (ColumnIsGeneratedStored(relationId, columnName))
```

Comment:
Why don't we check `if (attriuteForm-> attgenerated == ATTRIBUTE_GENERATED_STORED)`?

Reply:
oh true, we don't even need `ColumnIsGeneratedStored` :)